exercises/1/src/components/HackerNews.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="hacker-news">
+    <h1>News List</h1>
+    <div v-for="item in sortedItems" :key="item.id">
+      <NewsItem v-bind:news-item="item" v-on:news_remove="onNewsRemove"/>
+    </div>
+    <NewsForm v-on:news_add="onNewsAdd"></NewsForm>
+  </div>
+</template>
+
+<script>
+import NewsItem from './NewsItem.vue';
+import NewsForm from './NewsForm.vue';
+
+export default {
+  name: 'HackerNews',
+  components: {NewsItem, NewsForm},
+  data() {
+    return {
+      items: [{
+        id: 0,
+        title: "Start Message",
+        voteCount: 0,
+      }],
+      id: 1
+    }
+  },
+  computed: {
+    sortedItems() {
+      let copiedItems = [...this.items];
+      copiedItems.sort((o1, o2) => o2.voteCount - o1.voteCount)
+      return copiedItems;
```

Comment:
⭐ for sorted properties - even as computed property! Nice!

In the future, you could shorten like this:

```suggestion
      return [...this.items].sort((o1, o2) => o2.voteCount - o1.voteCount);
```